src/unix/kqueue.c

```diff
@@ -99,6 +99,44 @@ int uv__io_fork(uv_loop_t* loop) {
 int uv__io_check_fd(uv_loop_t* loop, int fd) {
   struct kevent ev;
   int rc;
+  struct stat sb;
+#ifdef F_GETPATH
```

Comment:
I think you can declare the variable inside the if block where you use it an avoid the ifdef.

Reply:
But `sb` will be used anyway regardless of whether the `F_GETPATH` exists.

Reply:
I meant the path.

Reply:
I thought `libuv` uses ANSI C89 style that requires variables to be declared at the beginning of a scope?